daq_macros.py

```diff
@@ -3260,7 +3285,37 @@ def loop_center_xrec():
   #now try to get the loopshape starting from here
   return 1
 
-  
+def xrec_no_zebra(angle_start):
+  print(f'xrec_no_zebra{angle_start}')
+  beamline_lib.mvaDescriptor("omega", angle_start)
+  #yield from bps.mv(samplexyz.omega, angle_start)
+  for omega_target in range (angle_start, angle_start+360, 40):
+    #yield from bps.mv(samplexyz.omega, omega_target)
+    beamline_lib.mvaDescriptor("omega", omega_target)
+    logger.info(f'taking image at {omega_target}')
+    timeout = 5
+    # change image mode to single(0)
+    setPvDesc("lowMagImMode",0)
+    # start camera
+    setPvDesc("lowMagAcquire",1)
+    try:
+      with urllib.request.urlopen(daq_utils.lowMagCamURL, timeout=timeout) as response:
+        logger.info("xnz: read")
+        image_data = response.read()
+        logger.info("xnz: read")
+      with open(getBlConfig("visitDirectory")+"findloop_"+str(omega_target/40)+".jpg", "wb") as filename:
```

Comment:
```suggestion
      with open(getBlConfig("visitDirectory")+"findloop_"+str(omega_target//40)+".jpg", "wb") as filename:
```

Be sure you get an int.